logstash-core/lib/logstash/api/commands/stats.rb

```diff
@@ -0,0 +1,105 @@
+require "logstash/api/commands/base"
+require 'logstash/util/thread_dump'
+
+module LogStash
+  module Api
+    module Commands
+      class Stats < ::LogStash::Api::Commands::Base
```

Comment:
Yep, this can be shortened. I'll just make it 'Commands::Base' because 'Base' is so generic it can be a little confusing.